ga4gh/datarepo.py

```diff
@@ -110,15 +110,17 @@ def getNumReferenceSets(self):
         """
         return len(self._referenceSetIds)
 
-    def getOntology(self, name):
+    def getOntologyTermMapByName(self, name):
         """
-        Returns an ontology map by name
+        Returns an ontologyTermMap by name
         """
-        return self._ontologyNameMap.get(name, None)
+        if name not in self._ontologyNameMap:
+            raise exceptions.OntologyNameNotFoundException(name)
```

Comment:
Thanks, I realized that but wanted to make sure this point was not lost. We may need to be able to work with multiple sequence ontology mappings.